### PR TITLE
majandofu.com

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/102493

<details><summary>Screenshot:</summary>

Снимок экрана 2021-12-21 в 14 36 03

https://majandofu.com/ca-baccarat-scouter20

Снимок экрана 2021-12-21 в 14 36 19

Снимок экрана 2021-12-21 в 14 36 46

</details><br/>

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -125,6 +125,8 @@ tokyoloader.com##section > div[align="center"] > font[size="-1"][color="#404040"
 tripeditor.com##.ad-h-resp
 tripeditor.com##.post_ads_box
 tripeditor.com##.relatedAdBox
+majandofu.com###banner-spread
+majandofu.com##a[href^="https://bit.ly"]
```
Comment (zloyden) [inline JapaneseFilter/sections/specific.txt:129]: I don't see it. Can you show the element with opened devtools?

Comment (jellizaveta) [reply JapaneseFilter/sections/specific.txt:129]: https://majandofu.com/ca-baccarat-scouter20

<details><summary>Screenshot:</summary>

Снимок экрана 2021-12-22 в 11 07 30

Снимок экрана 2021-12-22 в 11 08 00

</details><br/>

**Review by zloyden — APPROVED** [review]

**Comment by Yuki2718** [comment]
@zloyden Could you see my pending comment:

These too:
```
majandofu.com###banner-footer > div.g
majandofu.com###banner-rectangle
majandofu.com##section[id^="banner-infeed"]
```
(`majandofu.com###banner-footer` causes false positive on the top page.)

**Comment by zloyden** [comment]
> Could you see my pending comment:

No, but I see this.
<details><summary>Screenshot:</summary>

image

</details><br/>

**Comment by Yuki2718** [comment]
> No, but I see this.

Oh, okay.

**Pull request merged**